homeassistant/components/point/__init__.py

```diff
@@ -50,6 +58,10 @@
 
 PLATFORMS = [Platform.BINARY_SENSOR, Platform.SENSOR]
 
+type PointConfigEntry = ConfigEntry[api.AsyncConfigEntryAuth]
+
+CONF_REFRESH_TOKEN = "refresh_token"
```

Comment:
It's defined here and in config_flow. Please move it to const.py and use it both places

Reply:
I've removed it completely as it is not referenced.